.github/workflows/CI.yml

```diff
@@ -65,5 +65,6 @@ jobs:
       with:
         files: '*.zip'
         tag_name: ${{ steps.prepare_artifacts.outputs.short_commit_sha }}
+        name: "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/}): ${tag_name}"
```

Comment:
Can you explain how this works?

Reply:
It *should* get the "GITHUB_REF" global variable, and strip it of everything except the branch name, based on what I can see.

I used this page as a reference (using the second option first, which caused an error):
https://www.techiediaries.com/get-current-branch-name-github-actions-github_ref-bash/

If I've misunderstood this, feel free to explain how it should work.

Reply:
I was confused because commands can't be run outside of `run:`, which is why that's running it in a step.

Reply:
You're probably right, but then how does one use it outside that `run:`? It seems kinda silly that they would show a full example for one option and not the other on that guide. :/

Reply:
You can't use it outside of run. There is no other option.

Reply:
There *is* another option, but I don't really have the option of doing it myself in a PR.

For your consideration:
https://github.com/xenia-canary/xenia-canary/runs/2512953209